oic.r.cred.raml

```diff
@@ -95,8 +118,19 @@ traits:
         description: The request is invalid.
 
   post:
+
```

Comment:
Oh really? I would agree with that then. Somehow CR-50 has escaped my gaze.

I'll remove rowner and push an update.